### PR TITLE
Improve readability

### PR DESCRIPTION
**Pull request by JuanPotato**

### CODE REVIEW

File: src/da2013.rs [hunk 1]
```diff
@@ -1,10 +1,11 @@
 use std::os::unix::io;
 
 use byteorder::{LittleEndian, WriteBytesExt};
+
 use nix;
 use nix::fcntl;
-use nix::sys::stat::Mode;
 use nix::unistd;
+use nix::sys::stat::Mode;
```
Comment (9ary) [inline src/da2013.rs:8]: Alphabetical order

File: src/da2013.rs [hunk 2]
```diff
@@ -48,19 +50,27 @@ impl Da2013 {
...
         buf.push(0);
+
         // Status
         buf.push(0);
+
         // Padding
         let mut zeros = vec![0; 3];
         buf.append(&mut zeros);
+
+        // Data
```
Comment (9ary) [inline src/da2013.rs:61]: I think adding this comment here is a bit too verbose, the ones I already have are mostly for the stuff that is not obvious just by looking at the code itself.

File: src/da2013.rs [hunk 3]
```diff
@@ -48,19 +50,27 @@ impl Da2013 {
...
         buf.write_u16::<LittleEndian>(command).unwrap();
         buf.write_u16::<LittleEndian>(request).unwrap();
         buf.write_u16::<LittleEndian>(arg0).unwrap();
         buf.write_u16::<LittleEndian>(arg1).unwrap();
+
         // Padding
         let mut zeros = vec![0; 76];
         buf.append(&mut zeros);
+
+        // Footer
```
Comment (9ary) [inline src/da2013.rs:71]: Same.

File: src/main.rs [hunk 4]
```diff
@@ -1,9 +1,10 @@
-use std::{env,path,process};
-
+#[macro_use]
+extern crate nix;
 extern crate byteorder;
 extern crate getopts;
 extern crate libudev;
-#[macro_use] extern crate nix;
+
+use std::{env,path,process};
```
Comment (9ary) [inline src/main.rs:7]: Not quite sure about this, it violates my usual order of imports which is standard library, external libraries then internal modules.

File: src/main.rs [hunk 5]
```diff
@@ -14,8 +15,7 @@ fn usage(progname: &str, opts: getopts::Options) {
 
 fn get_hidraw_node() -> Option<path::PathBuf> {
     let udev = libudev::Context::new().expect("Failed to create udev context");
-    let mut enumerator = libudev::Enumerator::new(&udev)
-        .expect("Failed to obtain udev enumerator");
+    let mut enumerator = libudev::Enumerator::new(&udev).expect("Failed to obtain udev enumerator");
```
Comment (9ary) [inline src/main.rs:18]: Doesn't fit in 99 chars. :P

File: src/main.rs [hunk 6]
```diff
@@ -1,9 +1,10 @@
-use std::{env,path,process};
-
+#[macro_use]
+extern crate nix;
```
Comment (9ary) [inline src/main.rs:2]: Missed this one, again, alphabetical order.

Comment (JuanPotato) [reply src/main.rs:2]: leeeeength

File: src/main.rs (hunk 5 above)
Comment (JuanPotato) [reply src/main.rs:18]: I dont even think I did that purposely

File: src/main.rs (hunk 4 above)
Comment (JuanPotato) [reply src/main.rs:7]: I think the standard is putting extern first and #macro_use and similar thing at the top

File: src/da2013.rs (hunk 1 above)
Comment (JuanPotato) [reply src/da2013.rs:8]: aight

**Pull request closed without merging**